htdocs/class/module.textsanitizer.php

```diff
@@ -528,7 +528,7 @@ public function nl2Br($text)
      */
     public function addSlashes($text)
     {
-        if (!@get_magic_quotes_gpc()) {
+        if (version_compare(PHP_VERSION, '7.4.0', '>=') OR !is_function('get_magic_quotes_gpc') OR !get_magic_quotes_gpc()) {
```

Comment:
The proposed code will not correctly reflect the state of `get_magic_quotes_gpc()` in PHP <= 5.3.x which could be true or false.  It would appear a cleaner solution would be:
`if (!function_exists('get_magic_quotes_gpc') || !get_magic_quotes_gpc()) {`

Reply:
Yes, I forgot to include older versions of PHP 💯 
I tested, it works